frontend/src/index.js

```diff
@@ -14,8 +10,24 @@ import config from '../config';
 * @return {Object}
 */
 export default function init(options) {
-  return new Plugin({
-    ...config,
+  const { state, subscribe } = options;
```

Comment:
I'd say it's better to register a subscription via the `extension-config` to enable listening to the `userDataReceived$`. You could use a common event to update the specific plugin instance.

Reply:
done